main.tf

```diff
@@ -432,8 +432,10 @@ module "entitycore_svc" {
     ) : (
     "https://staging.openbraininstitute.org/auth/realms/SBO/"
   )
-  s3_bucket_name = var.entitycore_svc_s3_bucket_name
-  image_url      = var.entitycore_svc_image_url
+  s3_bucket_name            = var.entitycore_svc_s3_bucket_name
+  s3_bucket_allowed_origins = ["*"]
```

Comment:
shouldn't this be equal to `entitycore_svc_s3_bucket_allowed_origins ` ?

Reply:
Thanks, I've missed that. Fixed now.